transaction-context/src/lib.rs

```diff
@@ -828,25 +810,23 @@ impl InstructionContext {
 #[derive(Debug)]
 pub struct BorrowedAccount<'a> {
     transaction_context: &'a TransactionContext,
-    instruction_context: &'a InstructionContext,
-    index_in_transaction: IndexOfAccount,
-    // Program accounts are not part of the instruction_accounts vector, and thus None
-    index_in_instruction_accounts: Option<IndexOfAccount>,
     account: RefMut<'a, AccountSharedData>,
+    instruction_account: InstructionAccount,
+    ix_program_account_idx: IndexOfAccount,
```

Comment:
Would prefer the long names we use everywhere else, so: `index_in_transaction_of_current_program`